indico/web/client/styles/partials/_boxes.scss

```diff
@@ -398,7 +402,9 @@ $i-box-padding: 10px;
     > li {
         @include border-top();
         padding: 10px;
-        position: relative;
+        display: flex;
```

Comment:
I think it's this change that breaks the alignment of the search result list and email management page

Reply:
I've refactored these alignments. The previous rules should be intact now.